src/router/index.js

```diff
@@ -97,6 +98,15 @@ const router = new Router({
     },
   ],
   scrollBehavior(to, from, savedPosition) {
+    if (to.meta.scroll) {
+      const element = document.querySelector('.router-link-exact-active');
+      if (element) {
+        element.scrollIntoView({ behavior: 'smooth' });
```

Comment:
I reviewed this locally and didn't like the scroll behavior here because it actually makes the page jumpy again.

Instead can we just `return {}` to avoid any `scrollBehavior`. Also can we rename the `scroll` key to `skipScrollBehavior`. So in the end, diff should look like something like this.


```diff
diff --git a/src/router/index.js b/src/router/index.js
index 5b3ef67..9ef9724 100644
--- a/src/router/index.js
+++ b/src/router/index.js
@@ -25,6 +25,7 @@ const router = new Router({
         meta: {
           period: period.type,
           hasBanner: true,
+          skipScrollBehavior: true,
         },
       })),
     },
@@ -97,6 +98,10 @@ const router = new Router({
     },
   ],
   scrollBehavior(to, from, savedPosition) {
+    if (to.meta.skipScrollBehavior) {
+      return {};
+    }
+
     return savedPosition || { x: 0, y: 0 };
   },
 });

```